src/v/cluster/metrics_reporter.cc

```diff
@@ -317,6 +326,7 @@ ss::future<> metrics_reporter::try_initialize_cluster_info() {
     boost::uuids::random_generator_mt19937 uuid_gen(mersenne_twister);
 
     _cluster_info.uuid = fmt::format("{}", uuid_gen());
+    vlog(clusterlog.info, "Generated cluster UUID {}", _cluster_info.uuid);
```

Comment:
Until the cluster UUID is harmonized, maybe let's be more specific here and call this cluster ID to avoid confusing this with the bootstrap cluster UUID

Reply:
I've clarified the logs in this file to not say uuid any more